msbuild/Xamarin.MacDev.Tasks/Tasks/Archive.cs

```diff
@@ -220,18 +237,14 @@ public override bool Execute ()
 				arInfo.Add ("ArchiveVersion", new PNumber (2));
 				arInfo.Add ("CreationDate", new PDate (Now.ToUniversalTime ()));
 				arInfo.Add ("Name", new PString (plist.GetCFBundleName () ?? plist.GetCFBundleDisplayName ()));
+				arInfo.Add ("SchemeName", new PString (plist.GetCFBundleName () ?? plist.GetCFBundleDisplayName ()));
 
 				if (!string.IsNullOrEmpty (ProjectGuid))
 					arInfo.Add ("ProjectGuid", new PString (ProjectGuid));
 
 				if (!string.IsNullOrEmpty (ProjectTypeGuids))
 					arInfo.Add ("ProjectTypeGuids", new PString (ProjectTypeGuids));
 
-				if (!string.IsNullOrEmpty (SolutionPath)) {
```

Comment:
@jstedfast I think this information is added for VS, so the Archive Manager can match this to the corresponding solution, right?

Reply:
Correct. I was just about to comment on this!